code/__HELPERS/icon_smoothing.dm

```diff
@@ -156,9 +156,13 @@
 		var/list/U = list()
 		if(fixed_underlay)
 			if(fixed_underlay["space"])
-				U += image('icons/turf/space.dmi', SPACE_ICON_STATE, layer=TURF_LAYER)
+				var/image/I = image('icons/turf/space.dmi', SPACE_ICON_STATE, layer=TURF_LAYER)
```

Comment:
Might want to make this mutable appearance and cache it higher up; check out TG's implementation of this.